src/pages/Bridge/BridgeOverview/components/IssueForm/IssueForm.tsx

```diff
@@ -270,9 +282,7 @@ const IssueForm = ({ requestLimits, dustValue, issueFee }: IssueFormProps): JSX.
                 size='large'
                 loading={transaction.isLoading}
               >
-                {hasEnoughGriefingCollateralBalance
-                  ? t('issue')
-                  : t('insufficient_token_balance', { token: griefingCollateralTicker })}
+                {t('issue')}
```

Comment:
This change is invalid - error handling is needed in case the user doesn't have enough of griefing collateral balance to pay for issue request.